resources/public/css/style.css

```diff
@@ -1,9 +1,68 @@
+.splash-page-content {}
+.splash-button .splash-button-content {height: 120px; width: 120px}
+.splash-button .svg-icon {height: 64px; width: 64px}
 
+@media (max-width: 767px)
+{.splash-button .svg-icon {height: 32px; width: 32px}
+.splash-button-title-prefix {display: none}
+.splash-button .splash-button-content {height: 60px; width: 60px; font-size: 10px}
+.legal-footer-parent {display: none}}
 
-/* remember to define visible focus styles! 
```

Comment:
Will this file need to be updated or kept in sync with something pre-css being rendered by `garden` in order to stay accurate? If so, that is a bad approach and we should consider this a stopgap.